astropy/io/votable/connect.py

```diff
@@ -179,3 +180,103 @@ def write_table_votable(
 io_registry.register_reader("votable", Table, read_table_votable)
 io_registry.register_writer("votable", Table, write_table_votable)
 io_registry.register_identifier("votable", Table, is_votable)
+
+
+# VOTable with embedded/linked Parquet file #
+def write_table_votable_parquet(input, output, column_metadata, *, overwrite=False):
+    """
+
+    This function allows writing a VOTable (XML) with PARQUET
+    serialization. This functionality is currently not
+    supported by Astropy (with the reason that this method
+    requires writing multiple files: a VOTable/XML and
+    PARQUET table). This function presents a wrapper, which
+    allows to do this. The concept is simple and probably
+    can be improved substantially. We first save the PARQUET
+    table using Astropy functionality. Then, we create a
+    VOTable with binary serialization. The latter is modified
+    later to include an external reference to the create
+    PARQUET table file.
+
+    Parameters
+    ----------
+    input : `~astropy.table.Table`
+        The table to write out.
+
+    output : str
+        The filename to write the table to.
+
+    column_metadata : dictionary
+        Contains the metadata for the columns such as "unit" or
+        "ucd" or "utype".
+        (Example: {"id": {"unit": "", "ucd": "meta.id", "utype": "none"},
+                   "mass": {"unit": "solMass", "ucd": "phys.mass", "utype": "none"}})
```

Comment:
Might render better like this (unconfirmed):

```suggestion
        "ucd" or "utype". For example::

            {"id": {"unit": "", "ucd": "meta.id", "utype": "none"},
             "mass": {"unit": "solMass", "ucd": "phys.mass", "utype": "none"}}

```

Reply:
This is not in the public API docs, so honestly, it shouldn't matter. I for one like the original as it's more compact.